### ISSUE TITLE
True boolean for multiatlas or string boolean ?

### ISSUE THREAD

**Issue by nboisteault** [3liz/lizmap-plugin#176]
## Description
Today .cfg contain "True" or "False" for boolean which we'll fix in Lizmap 4. Should we keep this way to do until Lizmap 4 ?

@rldhont @mdouchin Thoughts ?

**Comment by mdouchin** [comment]
we should keep the old and bad way, until we change it for Lizmap 4

**Comment by Gustry** [comment]
Yes, we should switch one day to a proper JSON with boolean as bool.

I made a kind of hack in the multiatlas branch.
If a single atlas was setup, it was generating the same CFG as before, with boolean as strings. But if it's a multi atlas, as it's a new data structure, I was at the same time using proper boolean in the new config. This doesn't break existing config with existing LWC. But yes maybe @mdouchin is right and we should postpone to Lizmap 4 this kind of change.

**Comment by Gustry** [comment]
Ok, so from this comment https://github.com/3liz/lizmap-plugin/pull/170#issuecomment-557162960 I will rollback the change.

**Comment by Gustry** [comment]
Some cleaning as been done in e71d9b2298905fdd0c11e03b0c61a48e35b816ef and cd423b64f8629f68e0460953146c1802aa25d9d3
